server/commands/userCreator.test.ts

```diff
@@ -37,6 +38,43 @@ describe("userCreator", () => {
     expect(isNewUser).toEqual(false);
   });
 
+  it("should add authentication provider to existing user", async () => {
+    const team = await buildTeam({ inviteRequired: true });
+    const teamAuthProviders = await team.$get("authenticationProviders");
+    const authenticationProvider = teamAuthProviders[0];
+
+    const email = "mynam@email.com";
+    const existing = await buildUser({
+      email,
+      teamId: team.id,
+      lastActiveAt: null,
```

Comment:
By setting `lastActiveAt` to `null` here you're essentially creating an invite, so it's not testing what you're wanting. Should just `buildUser` and let it create an existing authentication too?